tox.ini

```diff
@@ -1,20 +1,100 @@
 [tox]
-envlist = py38, py39, py3.10, py311, py312, pypy3
+envlist = pypy3,py3{9,10,11,12,13}-{linux,macos,windows}
```

Comment:
I'm fairly sure a machine where you run tox can only have one OS loaded at a time. Is this needed?

Reply:
That ^^ matrix is how tox constructs env commands, including PLATFORM matrix.

Reply:
What is the PLATFORM matrix for?